packages/addresses/README.md

```diff
@@ -60,3 +60,24 @@ import { getBase58EncodedAddressFromPublicKey } from '@solana/addresses';
 
 const address = await getBase58EncodedAddressFromPublicKey(publicKey);
 ```
+
+### `getProgramDerivedAddress()`
+
+Given a program's `Base58EncodedAddress` and up to 16 `Seeds`, this method will return the program derived address (PDA) associated with each.
+
+```ts
+import { getBase58EncodedAddressCodec, getProgramDerivedAddress } from '@solana/addresses';
+
+const { serialize } = getBase58EncodedAddressCodec();
+const { bumpSeed, pda } = await getProgramDerivedAddress({
+    programAddress: 'ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL' as Base58EncodedAddress,
+    seeds: [
+        // Owner
+        serialize('9fYLFVoVqwH37C3dyPi6cpeobfbQ2jtLpN5HgAYDDdkm' as Base58EncodedAddress),
```

Comment:
Does this return the same bytes as `new PublicKey('9fYLFVoVqwH37C3dyPi6cpeobfbQ2jtLpN5HgAYDDdkm').toBuffer()` in the old lib?

Reply:
It does!

```ts
> new PublicKey('9fYLFVoVqwH37C3dyPi6cpeobfbQ2jtLpN5HgAYDDdkm').toBuffer()
<Buffer 80 bc fe 45 2a 9a 00 99 12 7e 5e de 0f 4f 69 98 01 18 0f a0 75 af c9 f5 df 28 ec 9b 28 c0 5d ba>
```

```ts
> Buffer.from(getBase58EncodedAddressCodec().serialize('9fYLFVoVqwH37C3dyPi6cpeobfbQ2jtLpN5HgAYDDdkm'))
<Buffer 80 bc fe 45 2a 9a 00 99 12 7e 5e de 0f 4f 69 98 01 18 0f a0 75 af c9 f5 df 28 ec 9b 28 c0 5d ba>
```